projects/plugins/jetpack/modules/markdown/easy-markdown.php

```diff
@@ -378,24 +407,30 @@ public function add_default_post_type_support() {
 
 	/**
 	 * Figure out the post type of the post screen we're on
+	 *
 	 * @return string Current post_type
 	 */
 	protected function get_post_screen_post_type() {
 		global $pagenow;
-		if ( 'post-new.php' === $pagenow )
-			return ( isset( $_GET['post_type'] ) ) ? $_GET['post_type'] : 'post';
-		if ( isset( $_GET['post'] ) ) {
-			$post = get_post( (int) $_GET['post'] );
-			if ( is_object( $post ) && isset( $post->post_type ) )
-				return $post->post_type;
+		$post_type = filter_var( INPUT_GET, 'post_type', FILTER_SANITIZE_STRING );
+		$post_id   = filter_var( INPUT_GET, 'post', FILTER_SANITIZE_NUMBER_INT );
```

Comment:
This does not seem correct. Should it have been `filter_input()` instead of `filter_var()`?

Reply:
Woops, nice catch - updated here: https://github.com/Automattic/jetpack/pull/22515#issuecomment-1022676789